src/magento/magento-marketplace-credentials.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Set Up Your Marketplace Credentials
+group: getting-started
+---
+
+Before you can install an extension purchased from Magento Marketplace, sign in to your Magento account and verify that you have an active access key. You can sign in to your Magento account from the header of [Marketplace][1] or [Magento.com][2].
+
+Your access key is a set of public and private keys that is used to synchronize your Magento installation with your Magento account, and verify your credentials. After your account is synchronized, will be reqired to enter your private key each time you install an extension or module from Magento Marketplace, or upgrade your Magento installation. You can create multiple access keys for different purposes, and enable or disable them as needed.
+
+## Step 1: Create an Access Key
+
+1. Sign in to your Magento account.
+
+1. On the _My Account_ page, choose the **Marketplace** tab.
+
+1. In the upper-right corner next to your name, click the down arrow and choose **My Profile**.
+
+![]({% link images/images/magento-marketplace-profile.png %}){: .zoom}
+_Your Marketplace Profile_
+
```

Comment:
This numbered section is not nested correctly and the numbering is not working correctly on the page in the staging build.

Reply:
Fixed.